content/kubernetes/re-clusters/delete_custom_resources.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: Delete custom resources
+linkTitle: Delete custom resources
+description: This article explains how to delete Redis Enterprise clusters and Redis Enterprise databases from your Kubernetes environment.
+weight: 90
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-clusters/delete_custom_resources.md,
+    /kubernetes/re-clusters/delete_custom_resources/,
+]
+---
+
+## Delete a database
+
+To delete a database managed by the Redis Enterprise Kubernetes operator, run `kubectl delete redb <your-db-name>` from your K8s cluster.
+
+When you delete a database, your data and the REDB custom resource file are also deleted.
+
+### Troubleshoot REDB deletion
+
+After a deletion request, the operator attaches a finalizer to the Redis Enterprise database (REDB) object. This makes sure the database is deleted before the REDB custom resource is removed from the K8s cluster.
```

Comment:
The finalizer is typically attached upon the creation of the object, and not upon its deletion. It also doesn't ensure the underlying database is deleted, but merely enables it. Here's a possible rewording:
```suggestion
The operator attaches a finalizer to the Redis Enterprise Database (REDB) object which allows the underlying database to be deleted before the REDB custom resource is removed from the K8s cluster.
```